language-plutus-core/src/Language/PlutusCore/Lexer.x

```diff
@@ -74,6 +74,8 @@ tokens :-
     <0> subtractInteger          { mkBuiltin SubtractInteger }
     <0> multiplyInteger          { mkBuiltin MultiplyInteger}
     <0> divideInteger            { mkBuiltin DivideInteger }
+    <0> quotientInteger          { mkBuiltin QuotientInteger }
+    <0> modInteger               { mkBuiltin ModInteger }
```

Comment:
This is nitpicking, but `remainderInteger` is sometimes in between `quotientInteger` and `modInteger` and sometimes goes after them.